### ISSUE TITLE
Quantification summary workflow: add percent unmatched

### ISSUE THREAD

**Issue by faricazjj** [iRNA-COSI/APAeval#327]

**Comment by dominikburri** [comment]
Hi, I'm currently on it implementing the metric, but I'm not sure how to exactly compute it. 
I get the number of matched sites and unmatched sites from `match_with_gt`, but it is unclear whether I should use all unmatched sites, that is unmatched prediction and ground truth sites or only one of them. Because this influences the result. 
An alternative could be to have a specific version based on the return df type in which the correlation is computed only for unmatched, matched prediction or ground truth sites (see #325). This would mean that this metric would also depend on that parameter.

Also, I'm planning to also include performance metrics from identification, in particular sensitivity and precision, because it is based off the same numbers. 
So, based on `summary_workflows/identification/specification/I_benchmark_specification.md` TP corresponds to nr of matched sites, FP to nr. of sites predicted but not found in ground truth and FN to nr. of sites not predicted but in ground truth.

**Comment by ninsch3000** [comment]
Would it make sense to report TPR,FPR, FNR (?) instead of percent unmatched then?

**Comment by ninsch3000** [comment]
Yes, of course. I just meant for example that TPR is the same as percent unmatched, if we're talking about taking only unmatched prediction.

**Comment by dominikburri** [comment]
> Yes, of course. I just meant for example that TPR is the same as percent unmatched (TP/(TP+FN)), if we're talking about taking only unmatched prediction.

yeah it would be the inverse and as percentage in that case. The question is do we only consider unmatched prediction or not.
